src/main/java/no/nav/familie/integrasjoner/client/rest/SkyggesakRestClient.kt

```diff
@@ -0,0 +1,57 @@
+package no.nav.familie.integrasjoner.client.rest
+
+import no.nav.familie.http.client.AbstractPingableRestClient
+import no.nav.familie.integrasjoner.felles.OppslagException
+import no.nav.familie.integrasjoner.sak.Skyggesak
+import no.nav.familie.log.mdc.MDCConstants
+import org.slf4j.LoggerFactory
+import org.slf4j.MDC
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.stereotype.Component
+import org.springframework.web.client.HttpClientErrorException
+import org.springframework.web.client.HttpStatusCodeException
+import org.springframework.web.client.RestClientException
+import org.springframework.web.client.RestOperations
+import java.net.URI
+
+@Component
+class SkyggesakRestClient(@Value("\${SKYGGE_SAK_URL}") private val skyggesakUrl: String,
+                          @Qualifier("sts") private val restTemplate: RestOperations)
+    : AbstractPingableRestClient(restTemplate, "skyggesak.sak") {
+
+    override val pingUri: URI = URI.create("$skyggesakUrl/internal/alive")
+    private val sakUri = URI.create("$skyggesakUrl/v1/saker")
+
+    private val logger = LoggerFactory.getLogger(SkyggesakRestClient::class.java)
+
+    fun opprettSak(request: Skyggesak) {
+        try {
+            postForEntity<Skyggesak>(sakUri, request, httpHeaders())
```

Comment:
Med tanke på att id i Skyggesak er readOnly(?), burde denne returnere id fra saken som blir opprettet? Eller hva brukes id til?

Reply:
Jeg tenkte på det, men 409 conflict returnerer ikke id. Så da måtte jeg evt gjort et søk for å hente id og akkurat nå har man ikke behov for den.

Reply:
men er jo ikke i bruk. Kanskje det er best å fjerne den?

Reply:
Ja, jeg tenker det er greit. Burde Skyggesak flyttes til kontrakter?